System/Process.hs

```diff
@@ -725,8 +729,6 @@ waitForProcess ph@(ProcessHandle _ delegating_ctlc _) = lockWaitpid $ do
             OpenExtHandle ph' job' -> do
               closePHANDLE ph'
               closePHANDLE job'
-              when delegating_ctlc $
```

Comment:
Yeah, see also the commit message:

> Additionally, remove endDelegateControlC entirely from the Windows-only
> OpenExtHandle branch, where it's a no-op and was also in the wrong place.

It seemed wrong to leave the misleading previous version, but also weird to write some
no-op version of the fix here since `delegating_ctlc` has no effect on Windows.

Reply:
I think an explicit comment in the code explaining why it's not needed would be a good thing. Otherwise someone may think it was just an oversight and add it back in the future.

Reply:
this is done now